src/libraries/System.Private.CoreLib/src/System/Threading/Interlocked.cs

```diff
@@ -126,6 +144,26 @@ public static UIntPtr Exchange(ref UIntPtr location1, UIntPtr value)
         public static ulong CompareExchange(ref ulong location1, ulong value, ulong comparand) =>
             (ulong)CompareExchange(ref Unsafe.As<ulong, long>(ref location1), (long)value, (long)comparand);
 
+        /// <summary>Compares two single-precision floating point numbers for equality and, if they are equal, replaces the first value.</summary>
+        /// <param name="location1">The destination, whose value is compared with <paramref name="comparand"/> and possibly replaced.</param>
+        /// <param name="value">The value that replaces the destination value if the comparison results in equality.</param>
+        /// <param name="comparand">The value that is compared to the value at <paramref name="location1"/>.</param>
+        /// <returns>The original value in <paramref name="location1"/>.</returns>
+        /// <exception cref="NullReferenceException">The address of <paramref name="location1"/> is a null pointer.</exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
AggressiveInlining wasn't shown necessary in my first explorations, added just for consistency with uint/ulong ones.